api/api.proto

```diff
@@ -654,19 +644,76 @@ message StorageCluster {
   repeated StorageNode nodes = 4;
 }
 
-// Enumerate
+service OpenStorageCluster {
+	// Enumerate lists all the nodes in the cluster.
+	rpc Enumerate(ClusterEnumerateRequest)
```

Comment:
nit: indentation

Reply:
Weird, I'm not sure what is going on there with the tabs. I'll check it.